Makefile

```diff
@@ -11,10 +11,6 @@ TESTNET_FLAGS ?=
 
 VERSION := $(shell echo $(shell git describe --tags 2>/dev/null ) | sed 's/^v//')
 COMMIT := $(shell git log -1 --format='%H')
-NETWORK ?= mainnet
-COVERAGE ?= coverage.txt
-BUILDDIR ?= $(CURDIR)/build
-LEDGER_ENABLED ?= true
```

Comment:
don't understand how these relate to the release file naming?

Reply:
I think he just conveniently remove several duplicated lines, not related to the subject.

Reply:
Yes, I conveniently removed several duplicated lines